spec/raven/event_spec.rb

```diff
@@ -347,6 +347,7 @@ def raven_context
   end
 
   describe '.initialize' do
+    # NOTE: This test is completely pointless.
```

Comment:
Btw a negative test would make it more obvious why there's a need for a `raise` matcher. (and this is a test coverage miss atm)